plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GesturesPluginImpl.kt

```diff
@@ -1231,6 +1229,10 @@ class GesturesPluginImpl : GesturesPlugin, GesturesSettingsBase {
       return false
     }
 
+    if (isPointAboveHorizon(ScreenCoordinate(e2.x.toDouble(), e2.y.toDouble()))) {
```

Comment:
nit:
extension for `MotionEvent`
```suggestion
    if (isPointAboveHorizon(e2.toScreenCoordinate())) {
```